ui/app/AppLayouts/Wallet/views/AccountContextMenu.qml

```diff
@@ -66,21 +67,21 @@ StatusMenu {
     }
 
     StatusAction {
-        objectName: "AccountMenu-AddNewAccountAction_" + root.account.name
+        objectName: "AccountMenu-AddNewAccountAction_" + root.name
         text: qsTr("Add new account")
-        enabled: !root.account
+        enabled: !root.address
         icon.name: "add"
         onTriggered: {
             root.addNewAccountClicked()
         }
     }
 
     Loader {
```

Comment:
Please remove the loader here. It's not necessary and makes that component not working fully properly (I know, enabling/disable that flag is not a case in practice, but it causes adding the same item multiple times).

![Screenshot from 2024-09-10 10-19-09](https://github.com/user-attachments/assets/b4064392-9870-4610-a053-27dea7377175)